ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -90,6 +90,11 @@ export class Send extends IronfishCommand {
       default: false,
       description: 'Allow offline transaction creation',
     }),
+    notes: Flags.string({
+      char: 'n',
+      description: 'The notes to include in the transaction',
```

Comment:
nit - maybe something like `note hashes to include`? just to be a little more explicit about what needs to be passed here

Reply:
yeah note hashes would be more explicit